Dockerfile-alpine

```diff
@@ -1,10 +1,12 @@
 FROM alpine:3.10
 LABEL maintainer "Lucas Lorentz <lucaslorentzlara@hotmail.com>"
```

Comment:
lol what the heck, nevermind... I typed this review comment on my phone when waking up, I think it might be a bug in the github mobile app that was hiding `< >` strings 😱

Reply:
Proof that I'm not insane 😂: 

![image](https://user-images.githubusercontent.com/2111701/82151881-dbfc9280-982b-11ea-82f0-28a4c06de8a1.png)